.cypress/integration/app_analytics_test/app_analytics.spec.js

```diff
@@ -432,11 +435,12 @@ describe('Viewing application', () => {
     cy.get('[data-test-subj="app-analytics-configTab"]').click();
     cy.get('select').select(visOneName);
     cy.intercept('PUT', `**/api/observability/application`).as('selectUpdate');
-    cy.wait('@selectUpdate')
-    cy.wait(2000); // despite the previous wait grabbing the call that updates the panel select, it doesn't appear without this
```

Comment:
Do we know why the tests now work without this?

Reply:
The test now reloads the home page after changing the availability which also makes the applications update properly